tests/openshift/ocp_node_recycle_test.go

```diff
@@ -0,0 +1,74 @@
+package tests
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/node"
+	. "github.com/portworx/torpedo/tests"
+	"github.com/sirupsen/logrus"
+)
+
+func TestOCPRecylceNode(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_recycle.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo : Recycle", specReporters)
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+// Sanity test for OCP Recycle method
+var _ = Describe("{RecycleOCPNode}", func() {
+
+	It("has to delete a node and wait for new node to be ready", func() {
+
```

Comment:
add the check that this test is running  OCP cluster

Reply:
This is not needed because when providing the other scheduler than "openshift", RecycleNode method throw unsupported error.
`
  Failed to recycle a node [ocp-airgapfafbst02-q2v4g-worker-fvqdh]. Error: [Function RecycleNode() is not supported]
  Unexpected error:
      <*errors.ErrNotSupported | 0xc0011f61a0>: {
          Type: "Function",
          Operation: <string>"RecycleNode()",
      }
      Function RecycleNode() is not supported
  occurred

`